spec/decorators/service_provider_session_decorator_spec.rb

```diff
@@ -71,6 +71,15 @@
           to be_nil
       end
     end
+
+    context 'sp has a blank custom alert' do
+      let(:sp) { build_stubbed(:service_provider, :with_blank_help_text) }
+
+      it 'uses the custom template' do
```

Comment:
let's fix the comment? it "is nil" maybe? or "it doesn't show anything"?

Reply:
oh wow, didn't even notice. good catch, updated